rero_ils/dojson/utils.py

```diff
@@ -1763,11 +1763,20 @@ def update_part(self, value_data, subfield_code, subfield_data):
         :param subfield_data: part number or name depending of `subfield_code`
         :type subfield_data: str
         """
+        def remove_last_dot(value):
+            """Remoces last dot from value."""
+            if value.count('.') == 1:
+                value = value.rstrip('.')
+            return value
```

Comment:
maybe
```
if value[-1] == '.':
  return value[:-1]
```

Reply:
@Garfield-fr This is not the purpose of this function if you read the docstring ;-)